### PR TITLE
kops: 1.4.1 (fix) 

### PR DESCRIPTION
**Pull request by WillPlatnick**
```kops version``` uses an ENV variable that needs to be present at build to denote the version. Without this, people don't know the version they are on.

### CODE REVIEW

File: Formula/kops.rb [hunk 1]
```diff
@@ -5,17 +5,11 @@ class Kops < Formula
   sha256 "69b3c9d7e214109cfd197031091ed23963383c894e92804306629f6a32ab324b"
   head "https://github.com/kubernetes/kops.git"
 
-  bottle do
-    cellar :any_skip_relocation
-    sha256 "99fc900bb11b242b4d3eca456fc7956233a8efa92f3dee7b321a005de0e94a28" => :sierra
-    sha256 "ed71d71b5031e0918478dec06b9064cf7e3f5b907128e98ec85a187801a27f8e" => :el_capitan
-    sha256 "0eee45caca5eb2a67ab88a90f9da226e99538d5198a196067b2a224a816bd6e0" => :yosemite
-  end
```
Comment (MikeMcQuaid) [inline Formula/kops.rb]: Please leave this as-is, it'll be updated when we pull.

File: Formula/kops.rb [hunk 2]
```diff
@@ -5,17 +5,11 @@ class Kops < Formula
...
-
   depends_on "go" => :build
   depends_on "kubernetes-cli"
 
   def install
+    ENV["VERSION"] = "1.4.1" unless build.head?
```
Comment (MikeMcQuaid) [inline Formula/kops.rb]: `ENV["VERSION"] = version`.

What should the behaviour be if it's building HEAD?

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

File: Formula/kops.rb (hunk 1 above)
Comment (WillPlatnick) [inline Formula/kops.rb]: Fixed.

File: Formula/kops.rb (hunk 2 above)
Comment (WillPlatnick) [inline Formula/kops.rb]: The makefile will grab the git commit for HEAD and set the environment variable appropriately.

**Comment by WillPlatnick** [comment]
@MikeMcQuaid Squashed and committed, I think that should fix everything raised.

**Comment by WillPlatnick** [comment]
@MikeMcQuaid Alrighty, using the version variable now.

**Comment by nijikon** [comment]
You are awesome, thanks for your contribution! :tada:

**Pull request closed without merging**